make/autoconf/flags-cflags.m4

```diff
@@ -809,6 +809,7 @@ AC_DEFUN([FLAGS_SETUP_CFLAGS_CPU_DEP],
       elif test "x$FLAGS_CPU" = xppc64le; then
         # Little endian machine uses ELFv2 ABI.
         # Use Power8, this is the first CPU to support PPC64 LE with ELFv2 ABI.
+        # After we switch to Power9+, we can use higher releases than Ubuntu 20.04 in class DockerfileConfig used for container tests
```

Comment:
Hm, my feeling is that we should not bloat the comments here. Maybe revert this line.